ui/src/app/edge/live/Multiple/Evcs_Api_Cluster/modal/evcs-chart/evcs.chart.ts

```diff
@@ -70,19 +70,18 @@ export class EvcsChartComponent implements OnInit, OnChanges {
         this.datasets[index].data = [chargePowerKW != null ? chargePowerKW : 0];
       }
       index++;
-    };
+    }
     this.loading = false;
   }
 
   getMaxPower() {
-    let maxPower: number;
-    let minPower = 22;
+    const minPower = 22;
     let maxHW = this.currentData[this.componentId + '/MaximumHardwarePower'];
     let chargePower = this.currentData[this.componentId + '/ChargePower'];
     maxHW = maxHW == null ? minPower : maxHW / 1000;
     chargePower = chargePower == null ? 0 : chargePower / 1000;
 
-    maxPower = chargePower < minPower || maxPower < minPower ? minPower : maxHW;
+    const maxPower: number = chargePower < minPower || maxHW;
```

Comment:
working great 👍

Reply:
memo: for easy understanding

original sentense use `maxPower` without initialization so it assumed as `undefined`.
so the sentence will evaluated following steps

1. maxPower = chargePower < minPower || maxPower < minPower ? minPower : maxHW;
2. maxPower = chargePower < minPower || undefined < minPower ? minPower : maxHW;
3. maxPower = chargePower < minPower || false ? minPower : maxHW;
4. maxPower = chargePower < minPower || maxHW;

so I rewrited by final sentense.